package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.2.21",
+  "version": "5.3.0",
```

Comment:
Patch number 21 was getting pretty high and this is a significant change to what is correctable, so I took this opportunity to increase the minor version.